### PR TITLE
SSCS-10895 remove PO attending from override fields

### PR DESCRIPTION
**Pull request by nickhill111**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

https://tools.hmcts.net/jira/browse/SSCS-10895

Updating tribunals so it can now deal with the override fields without poToAttend

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

**Review by bensonlievoco — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/updatelistingrequirements/UpdateListingRequirementsAboutToStartHandler.java [hunk 1]
```diff
@@ -33,26 +34,36 @@ public boolean canHandle(CallbackType callbackType, Callback<SscsCaseData> callb
...
-                sscsCaseData.getSchedulingAndListingFields().setOverrideFields(overrideFields);
+                schedulingAndListingFields.setOverrideFields(overrideFields);
+            }
+
+            OverrideFields listingValues = schedulingAndListingFields.getDefaultListingValues();
+
+            if (isNull(listingValues)) {
+                listingValues = new OverrideFields();
+                schedulingAndListingFields.setDefaultListingValues(listingValues);
             }
```
Comment (TeaWithLucas) [inline src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/updatelistingrequirements/UpdateListingRequirementsAboutToStartHandler.java:67]: Why is this here? DefaultListingValues can be left as null until it is set in hearings api

**Review by TeaWithLucas — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/updatelistingrequirements/UpdateListingRequirementsAboutToStartHandler.java (hunk 1 above)
Comment (nickhill111) [reply src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/updatelistingrequirements/UpdateListingRequirementsAboutToStartHandler.java:67]: Removed

**Review by TeaWithLucas — APPROVED** [review]

**Review by bensonlievoco — APPROVED** [review]

**Review by hgulv1 — APPROVED** [review]

**Review by EliandroGV1 — APPROVED** [review]

File: src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/hmctsresponsereviewed/HmctsResponseReviewedAboutToSubmitHandlerTest.java [hunk 2]
```diff
@@ -48,13 +46,14 @@ public class HmctsResponseReviewedAboutToSubmitHandlerTest {
...
-        dwpDocumentService = new DwpDocumentService();
+        DwpDocumentService dwpDocumentService = new DwpDocumentService();
         handler = new HmctsResponseReviewedAboutToSubmitHandler(dwpDocumentService);
 
         when(callback.getEvent()).thenReturn(EventType.HMCTS_RESPONSE_REVIEWED);
         when(callback.getCaseDetails()).thenReturn(caseDetails);
         sscsCaseData = SscsCaseData.builder().ccdCaseId("ccdId").appeal(Appeal.builder().build())
-                .selectWhoReviewsCase(new DynamicList(new DynamicListItem("reviewByTcw", "Review by TCW"), null))
+                .selectWhoReviewsCase(new DynamicList(
+                    new DynamicListItem("reviewByTcw", "Review by TCW"), null))
```
Comment (hgulv1) [inline src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/hmctsresponsereviewed/HmctsResponseReviewedAboutToSubmitHandlerTest.java:56]: in the future avoid making changes where not relevant as you've made alot of formatting changes to this file it bloats the PR

File: src/main/java/uk/gov/hmcts/reform/sscs/util/UpdateListingRequirementsUtil.java [hunk 3]
```diff
@@ -78,70 +60,4 @@ private DynamicListItem getLanguageDynamicListItem(Language language) {
...
-                .map(UpdateListingRequirementsUtil::getJudicialMemberType)
-                .findFirst()
-                .orElse(null);
-            if (judicialMemberType != null) {
-                return judicialMemberType.getHmcReference();
-            }
-
-        }
-        return StringUtils.EMPTY;
-    }
```
Comment (hgulv1) [inline src/main/java/uk/gov/hmcts/reform/sscs/util/UpdateListingRequirementsUtil.java:146]: Why were all these methods removed?

**Review by hgulv1 — CHANGES_REQUESTED** [review]

File: src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/hmctsresponsereviewed/HmctsResponseReviewedAboutToSubmitHandlerTest.java (hunk 2 above)
Comment (nickhill111) [reply src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/hmctsresponsereviewed/HmctsResponseReviewedAboutToSubmitHandlerTest.java:56]: Personally I think its good to cleanup files which you work in, it cleans up the codebase fairly well and means you always leave it in a nicer state than it was. Given that some of these files I didn't make any functional changes and I will be mindful of changing them in the future

**Review by hgulv1 — APPROVED** [review]

**Review by TeaWithLucas — APPROVED** [review]
